### PR TITLE
meld 3.12.3

### PR DESCRIPTION
**Pull request by tschoonj**
version bump
gtk+3 based now
audit --strict compliance

### CODE REVIEW

**Comment by tschoonj** [comment]
@DomT4 Any idea whats going on here? Is the PYTHONPATH variable not being set in the test block though I ask for it? The `gi` module should be available...

**Comment by tschoonj** [comment]
Ok never mind, figured out the problem eventually...

**Comment by DomT4** [comment]
> Is the PYTHONPATH variable not being set in the test block though I ask for it? 

As long as you do something like:

```ruby
test do
  ENV["PYTHONPATH"] = "example"
  system "example", "example"
end
```

Then it should be set.

File: meld.rb [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
-  depends_on 'pygobject'
+  depends_on "gtksourceview3"
+  depends_on "pygobject3"
+  depends_on "gobject-introspection"
 
   def install
-    system "make", "prefix=#{prefix}", "install"
-    bin.env_script_all_files(libexec+'bin', :PYTHONPATH => ENV['PYTHONPATH'])
+    ENV.append_path "PYTHONPATH", "#{Formula["libxml2"].opt_lib}/python2.7/site-packages"
+    system "python", "setup.py", "--no-update-icon-cache",
```
Comment (DomT4) [inline meld.rb]: Will this accept something like `system "python", "--no-update-icon-cache", "--no-compile-schemas", *Language::Python.setup_install_args(prefix)"`?

**Comment by tschoonj** [comment]
No that doesn't work: the icon-cache and schemas arguments have to come between setup.py and install, while prefix has to be last.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-x11/commit/ca2a8efacf02f15b112f999e10666a55b6fefd62. Thanks @tschoonj!

**Pull request closed without merging**